docker/scripts/start_template_renderer.sh

```diff
@@ -12,5 +12,6 @@ then
     sudo -E -H -u musicbrainz carton exec -- ./script/compile_resources.sh server
 fi
 
-exec sudo -E -H -u musicbrainz \
+export HOME="$MBS_HOME"
+exec chpst -u musicbrainz:musicbrainz \
     carton exec -- ./script/start_renderer.pl 2>&1
```

Comment:
This script (`docker/scripts/start_template_renderer.sh`) is also used in website images which aren’t using `runit` and thus don’t have `chpst`. This change should rather occur in `docker/musicbrainz-tests/template-renderer.service` which is running it in `musicbrainz-tests` image.

Maybe that will also require additional changes to make this script use `sudo` only when needed as in https://github.com/metabrainz/musicbrainz-server/blob/v-2024-07-22.0/admin/run#L46-L64 for example.

Reply:
> This script (`docker/scripts/start_template_renderer.sh`) is also used in website images which aren’t using `runit` and thus don’t have `chpst`.

The production website images do use runit, just not svlogd--is there another website image I'm missing?

Reply:
Oops, my bad, I probably checked a container running some other image.